Wikipedia/Code/SearchLanguagesBarViewController.swift

```diff
@@ -106,15 +108,8 @@ class SearchLanguagesBarViewController: UIViewController, WMFPreferredLanguagesV
     
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-        updateLanguageBarLanguageButtons()
-        
-        var selectedButtonCount = 0
-        for button in languageButtons{
-            if button.isSelected {
-                selectedButtonCount += 1
-            }
-        }
-        assert(selectedButtonCount == 1, "One button should be selected by now")
+        updateSearchLanguageButtons()
+        assert(searchLanguageButtons().filter { $0.isSelected }.count == 1)
```

Comment:
I'm not sure the coding style. Should the reason string be moved over as well? In this case it's not necessarily clear there should be a selection by the time `viewWillAppear()` is called, so it might be good to move it over as well.

Reply:
This is another case where I was just converting over what existed previously. I'm just going to remove this assertion. Updated!